e2e/tests/ui/features/@importer-explorer/importer-explorer.step.ts

```diff
@@ -0,0 +1,449 @@
+import { createBdd } from "playwright-bdd";
+import { ElementHandle, expect } from "playwright/test";
+import { DetailsPage } from "../../helpers/DetailsPage";
+import { ToolbarTable } from "../../helpers/ToolbarTable";
+import { SearchPage } from "../../helpers/SearchPage";
+import { test } from "@playwright/test";
```

Comment:
We have tweaked/extended 'test' object, so please replace with:
```suggestion
import { test } from "../../fixtures";
```